dashboard/scripts/test_seed

```diff
@@ -3,13 +3,11 @@
 # test the seed:all rake task (mostly that it is idempotent)
 
 SQL="select id, \`key\` from videos; \
-     select id, name, video_id from concepts; \
```

Comment:
That was my hope, though I wasn't aware of a video dependency. Let me not remove concepts for now. Will update PR when I get the chance, it may be tomorrow AM before I push a new commit.